server/testkit/src/lib.rs

```diff
@@ -102,25 +102,24 @@ pub async fn create_user(rsclient: &KanidmClient, id: &str, group_name: &str) {
         .expect("Failed to create the user");
 
     // Create group and add to user to test read attr: member_of
-    #[allow(clippy::expect_used)]
+    #[allow(clippy::panic)]
```

Comment:
Why panic over expect?

Reply:
I tried an `expect(format!(...))``, but clippy yelled at me and told me to go this way because of the [expect_fun_call](https://rust-lang.github.io/rust-clippy/master/index.html#/expect_fun_call) lint.